tensorflow/compiler/tf2tensorrt/segment/segment.cc

```diff
@@ -709,36 +796,43 @@ Status SegmentGraph(const Graph* tf_graph,
 
   // --------------------------------- Step 1 ---------------------------------
   auto graph = std::unique_ptr<SimpleGraph>(new SimpleGraph(tf_graph));
+
+  // Fetch the user-provide TF operations denylisted for conversion by TF-TRT.
+  const gtl::FlatSet<string> tftrt_op_denylist = [] {
+    string tftrt_op_denylist_str;
+    TF_CHECK_OK(ReadStringFromEnvVar("TF_TRT_OP_DENYLIST", /*default_value=*/"",
+                                     &tftrt_op_denylist_str));
+    gtl::FlatSet<string> tftrt_op_denylist{};
```

Comment:
I saw this error:
Use absl instead of tensorflow functions/types. Suppress with 'non-absl ok'.

I think you should use absl:flat_hash_set instead.

Reply:
Fixed